pages/views/index.11tydata.yml

```diff
@@ -0,0 +1,89 @@
+seoTitle: ESL — A Web Components Library
+description: >
+  ESL is an open-source UI component library based on web components.
+  Create user-friendly interfaces and build sites from the ground up faster using framework agnostic web components.
+
+landing:
+  marquee:
+    logo: logo-pretty.svg
+    title: Exadel Smart Library
+    slogan: UI component library with flair and precision
+    description: >
+      ESL is an open-source web components library that helps developers
+      create modern-looking, effective, and user-friendly interfaces faster.
+    ctas:
+      - text: Get started
+        url: /introduction
+        style: sec-blue
+      - text: See the library in action
+        url: /examples
+        style: sec-orange
+
+  benefits:
+    title: Why the ESL Web Components Library?
+    items:
+      - icon: benefits/efficient.svg
+        name: Efficient
+        text: >
+          Our team has reduced the work needed to create reusable UI components by giving you a reliable base
+          to customize and expand on for fast and accurate development.
```

Comment:
```suggestion
          Our team goal is to reduce development time by providing heavily customizable and expandable UI components
```

Reply:
It seems to me the current version more suits the marketing site.
Anyway, let's deliver such corrections to our content team proofreaders.